README.md

```diff
@@ -5,31 +5,31 @@
 
 #### Compose a cloud native masterpiece.
 
-Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub.
+Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub. To find out more about Appsody check out [appsody.dev](https://appsody.dev).
 
 # Stacks
 
 This repository holds the default set of stacks available from Appsody. Each stack is pre-configured with popular open source technologies to enable rapid development of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development.
 
 Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.
 
-To find out more about Appsody check out [appsody.dev](https://appsody.dev).
+Click here to find out more about [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview).
```

Comment:
For more information, see [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview).